app/views/licence-monitoring-station/setup/abstraction-period.njk

```diff
@@ -0,0 +1,115 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: backLink
+    })
+  }}
+{% endblock %}
+
+{# If there is an abstractionPeriodStart error, the error class and message are set #}
+{% if error.text.startResult %}
+  {% set startResultErrorClass = " govuk-input--error" %}
+  {% set startResultErrorMessage = { text: error.text.startResult } %}
+{% else %}
+  {% set startResultErrorClass = "" %}
+  {% set startResultErrorMessage = null %}
+{% endif %}
+
+{# If there is an abstractionPeriodEnd error, the error class and message are set #}
+{% if error.text.endResult %}
+    {% set endResultErrorClass = " govuk-input--error" %}
+    {% set endResultErrorMessage = { text: error.text.endResult } %}
+{% else %}
+  {% set endResultErrorClass = "" %}
+  {% set endResultErrorMessage = null %}
+{% endif %}
+
+{% block content %}
+{# Error summary #}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text:error.text.startResult,
+          href: "#abstraction-period-start"
+        },
+        {
+          text:error.text.endResult,
+          href: "#abstraction-period-end"
+        }
+      ]
+    }) }}
+  {%endif%}
+
+  {# Main heading #}
+  <span class="govuk-heading-l govuk-!-margin-bottom-3">
+    <span class="govuk-caption-l">{{ monitoringStationLabel }}</span> {{ pageTitle }}
+  </span>
+
+  <form method="post">
+    <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
+
+    {# From #}
+    {{ govukDateInput({
+      id: "abstraction-period-start",
+      namePrefix: "abstraction-period-start",
+      errorMessage: startResultErrorMessage,
+      fieldset: {
+        legend: {
+          text: "Start date",
+          classes: "govuk-fieldset__legend--xs govuk-!-font-weight-bold"
+        }
+      },
+      items: [
+        {
+          name: "day",
+          classes: "govuk-input--width-2" + startResultErrorClass,
```

Comment:
A suggestion but might need a wider team agreement we can just use inline if statements instead of setting variables: 

`classes: "govuk-input--width-2 {{ 'govuk-input--error' if error }}"`